schema/filter_artworks.js

```diff
@@ -48,6 +48,9 @@ const FilterArtworks = {
     dimension_range: {
       type: GraphQLString,
     },
+    extra_aggregation_gene_ids: {
```

Comment:
Assuming you agree with https://github.com/artsy/gravity/pull/9709 , this will let you pass in an arbitrary list of gene id's, and have them included in the medium aggregation response.